pymc/backends/report.py

```diff
@@ -34,7 +34,7 @@ def __init__(self) -> None:
 
     @property
     def _warnings(self):
-        chains = sum(self._chain_warnings.values(), [])
+        chains = [w for warnings in self._chain_warnings.values() for w in warnings]
```

Comment:
I don't find the two level list comprehension more readable than the sum concatenation tbh. Is this a default code style rule or personal preference?

Reply:
I prefer chain.from_iterable

Reply:
RUF017 suggests using `ft.reduce`, but indeed, `it.chain.from_iterable` looks like the best option: https://stackoverflow.com/questions/952914/how-do-i-make-a-flat-list-out-of-a-list-of-lists/953097#953097
I'll change everything.

Reply:
Is it really the double-comprehension that makes this code difficult to read?

Does using itertools actually make this easier to read?

I personally would have used something like: 

```python
warning_groups = self._chain_warnings.values()
chained_warnings = [warning for group in warning_groups for warning in group]
return chained_warnings + self._global_warnings
```

But it's all just style preference.

Reply:
double nested comprehension just to flatten is really odd for me. You are not filtering or doing anything with the items.

The purpose of the operation is much more obvious with list(chain.from_iterable(iterable))